src/maker/handlers.rs

```diff
@@ -300,8 +300,8 @@ impl Maker {
                 },
                 funding_output.value,
                 &funding_info.contract_redeemscript,
-                Amount::from_sat(message.next_fee_rate),
-            )?;
+                FeeRate::from_sat_per_vb(message.next_fee_rate),
```

Comment:
Instead of converting `u64` to  `FeeRate` -> directly annotate `next_fee_rate` field of `ProofOfFunding` as `FeeRate` type.
https://github.com/citadel-tech/coinswap/blob/67c39a9383fc1daff9705afe574cc75870379f3c/src/protocol/messages.rs#L145-L146